src/Blueprint/Blueprint/Az.Blueprint.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.2.0'
```

Comment:
@filizt you can revert the changes made in this file -- we're version the module before release